platform/src/main/java/org/hillview/storage/CsvGzReader.java

```diff
@@ -0,0 +1,144 @@
+package org.hillview.storage;
```

Comment:
It looks like we don't need this class at all.

Reply:
Correct, I thought I removed it from git, apparently I didn't.